Infrastructure.Data/Mapping/CustomerMapping.cs

```diff
@@ -0,0 +1,56 @@
+﻿using DomainModels.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Infrastructure.Data.Mapping
+{
+    internal class CustomerMapping : IEntityTypeConfiguration<Customer>
+    {
+        public void Configure(EntityTypeBuilder<Customer> builder)
+        {
+            builder.ToTable("customer");
+
+            builder.HasKey(customer => customer.Id);
+
+            builder.Property(customer => customer.Id)
+                .IsRequired()
+                .HasColumnName("id");
+            builder.Property(customer => customer.Email)
+                .IsRequired().
+                HasColumnName("Email");
```

Comment:
E-mail também pode ser único
```suggestion
            builder.Property(customer => customer.Email)
                .IsRequired()
                .HasColumnName("Email");
```